### PR TITLE
Add browserContext.addInitScript documentation

### PR DESCRIPTION
**Pull request by ankur22**
## What?

Adding `browserContext.addInitScript` documentation for the browser module, which was missing from our docs.

## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [x] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [x] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

## Related PR(s)/Issue(s)

NA since this was missing documentation.

### CODE REVIEW

**Comment by ankur22** [comment]
> LGTM 👍 Can you help me understand why we should add the same content to the earlier versions (v0.47.x etc.)?

`addInitScript` on `browserContext` has been around for quite a few version already, so i'm adding them to the earlier version of the doc too.

File: docs/sources/next/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: 'addInitScript()'
+description: 'Adds an init script.'
+---
+
+# addInitScript()
+
+Adds a script which will be evaluated in one of the following scenarios:
+
+- Whenever a page is created in the browser context or is navigated.
```
[the remark this answers is not in the document]
Comment (ankur22) [reply docs/sources/next/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md]: Sure, done in 33bbb146fc89d497d39ecaaf0ec2854024d0d1ff.

File: docs/sources/next/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Whenever a child frame is attached or navigated in any page in the browser context. In this case, the script is evaluated in the context of the newly attached frame.
```
[the remark this answers is not in the document]
Comment (ankur22) [reply docs/sources/next/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md]: Sure, done in 33bbb146fc89d497d39ecaaf0ec2854024d0d1ff.

File: docs/sources/next/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+# addInitScript()
+
+Adds a script which will be evaluated in one of the following scenarios:
+
+- Whenever a [page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) is created in the [browserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext) or is navigated.
+- Whenever a child [frame](<[page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/)>) is attached or navigated in any [page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) in the [browserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext). In this case, the script is evaluated in the context of the newly attached [frame](<[page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/)>).
+
+The script is evaluated after the document is created but before any of its scripts were run. This is useful to amend the JavaScript environment, e.g. to override `Math.random`.
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md:13]:
```suggestion
The script is evaluated after the document is created but before any of its scripts are run. This is useful to amend the JavaScript environment, for example, to override `Math.random`.
```

File: docs/sources/v0.47.x/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+# addInitScript()
+
+Adds a script which will be evaluated in one of the following scenarios:
+
+- Whenever a [page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) is created in the [browserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext) or is navigated.
+- Whenever a child [frame](<[page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/)>) is attached or navigated in any [page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) in the [browserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext). In this case, the script is evaluated in the context of the newly attached [frame](<[page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/)>).
+
+The script is evaluated after the document is created but before any of its scripts were run. This is useful to amend the JavaScript environment, e.g. to override `Math.random`.
```
Comment (heitortsergent) [inline docs/sources/v0.47.x/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md:13]:
```suggestion
The script is evaluated after the document is created but before any of its scripts are run. This is useful to amend the JavaScript environment, for example, to override `Math.random`.
```

File: docs/sources/v0.48.x/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+# addInitScript()
+
+Adds a script which will be evaluated in one of the following scenarios:
+
+- Whenever a [page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) is created in the [browserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext) or is navigated.
+- Whenever a child [frame](<[page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/)>) is attached or navigated in any [page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) in the [browserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext). In this case, the script is evaluated in the context of the newly attached [frame](<[page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/)>).
+
+The script is evaluated after the document is created but before any of its scripts were run. This is useful to amend the JavaScript environment, e.g. to override `Math.random`.
```
Comment (heitortsergent) [inline docs/sources/v0.48.x/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md:13]:
```suggestion
The script is evaluated after the document is created but before any of its scripts are run. This is useful to amend the JavaScript environment, for example, to override `Math.random`.
```

File: docs/sources/v0.49.x/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+# addInitScript()
+
+Adds a script which will be evaluated in one of the following scenarios:
+
+- Whenever a [page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) is created in the [browserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext) or is navigated.
+- Whenever a child [frame](<[page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/)>) is attached or navigated in any [page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) in the [browserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext). In this case, the script is evaluated in the context of the newly attached [frame](<[page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/)>).
+
+The script is evaluated after the document is created but before any of its scripts were run. This is useful to amend the JavaScript environment, e.g. to override `Math.random`.
```
Comment (heitortsergent) [inline docs/sources/v0.49.x/javascript-api/k6-experimental/browser/browsercontext/addinitscript.md:13]:
```suggestion
The script is evaluated after the document is created but before any of its scripts are run. This is useful to amend the JavaScript environment, for example, to override `Math.random`.
```

**Review by heitortsergent — APPROVED** [review]
Minor edit there, thanks for porting it to the older versions! 🙇

**Review by inancgumus — APPROVED** [review]

**Pull request merged**